lib/cc/engine/analyzers/violation.rb

```diff
@@ -91,19 +112,15 @@ def fingerprint
           digest << "-"
           digest << current_sexp.mass.to_s
           digest << "-"
-          digest << occurrences.to_s
+          digest << occurrences.length.to_s
```

Comment:
Thanks for your guys' input!

Sounds like there's a lot of philosophy to consider.

I think those things are worth investigating and discussing, but we could also make some of the small changes at the same time and incrementally move our sculpture towards the desired final shape - releasing the figure from within the marble block. (More philosophy ;) )

E.g. ship tuning points algorithm update and reporting issue for each duplication instance. 
Maybe thinking about an occurrence of duplication as many issues instead of one is fine?

Going to extract some of the tiny cuts into separate PRs.